apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/Operations.java

```diff
@@ -428,24 +426,6 @@ public void close() throws Exception {
     spark.close();
   }
 
-  private Catalog getCatalog() {
```

Comment:
this is refactoring to ensure this method is reused. 
In fact, we should consider publishing this method into an artifact, or is this available in the starter-kit ? @HotSushi

Reply:
yes! will vote on publishing this as part of spark-runtime/java-runtime whichever suitable. Starter-kit is non-existent as of now. We shouldnt also not add it there if we plan to support it first class.

Reply:
Let's not do that in this PR. Why did we need this refactor in this PR? dont see any `/src` usages

Reply:
The reason this is done is to avoid code duplication. IMO that's the worse comparing to moving this method into some module that we are going to publish as artifact.

Reply:
I don't see it getting re-used in this PR besides testCatalogWriteAPI. Are you provisioning for future pr/code copy?

Reply:
I am moving things into integration/spark-runtime module. Plz check if this is close to the middle ground.

Reply:
@HotSushi usage in `com.linkedin.openhouse.jobs.spark.Operations`